### PR TITLE
Python 3 forumfix

### PR DESCRIPTION
**Pull request by BobCratchett**
Fix as described on Kodi forums - https://forum.kodi.tv/showthread.php?tid=178294&pid=2907775#pid2907775

Note, this fix removes python 2/3 compatibility for this particular call, and has not been tested by me.

### CODE REVIEW

File: resources/lib/library.py [hunk 1]
```diff
@@ -2032,9 +2032,6 @@ def getImagesFromVfsPath(self, path):
                 if not image and item.get("file",""):
                     image = item["file"]
                 if image:
-                    if sys.version_info.major == 3:
-                        image = urllib.parse.unquote(image).decode('utf8')
-                    else:
```
Comment (AnonTester) [inline resources/lib/library.py]: the python3 check and the urllib.parse needs to be reinstated here, all that needs to be removed is the decode('utf8') from this line.

Comment (BobCratchett) [reply resources/lib/library.py]: Thanks for taking the time to review the PR. Cards on the table, I've absolutely f'ed up this PR, and done it against the wrong branch (I'm suggesting Python3 changes, but not against the Py3 branch of the script). I'll sort it so it's against the correct branch (including the change you suggest) in the new year, on the proviso you again review my code changes when they're actually against the correct branch :)

Comment (AnonTester) [reply resources/lib/library.py]: Don't worry about it. I figured out how to do PRs and have created a new PR with this fix as well as another small one. https://github.com/mikesilvo164/script.skinshortcuts/pull/13

Comment (BobCratchett) [reply resources/lib/library.py]: Fantastic. In that case, I'll happily close this one.

**Pull request closed without merging**